cmd/manifest/init-v2.go

```diff
@@ -42,10 +43,15 @@ import (
 	"github.com/spf13/cobra"
 )
 
+type analyticsTrackerInterface interface {
```

Comment:
should we move it to analytics package to hold this instead of declaring an interface in each command?

Reply:
I made this decision based on the current implementation of the AnalyticsTracker, which holds the methods to track different commands/events.

With this context, i preferred to declare when i needed the interface with just the methods i will need in that package, a benefit of this is that when mocking this on the tests i only take into account the method that is being used on this package from the whole implementation. Also it makes it more modular between interface/implementation, although the downsize is to declare the interface when the implementation is injected on every package.